src/interface/src/app/signup/signup.component.spec.ts

```diff
@@ -94,15 +94,4 @@ describe('SignupComponent', () => {
       );
     });
   });
-
-  describe('login', () => {
```

Comment:
Why are we removing the login test?

Reply:
The signup component no longer performs a login. Its only user-visible action is to send a validation email (although it does create the user account in the background – but that's all done on the backend, and I can't test it from here)